configuration/configuration.go

```diff
@@ -0,0 +1,96 @@
+package configuration
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	yaml "gopkg.in/yaml.v2"
+
+	"github.com/fsnotify/fsnotify"
+	"github.com/spf13/viper"
+)
+
+const (
+	// DefaultConfigFilePath is the path to the configuration file that is used if no other
+	// file is specified via the -config switch or via the ALMIGHTY_CONFIG_FILE_PATH environment
+	// variable.
+	DefaultConfigFilePath = "config.yaml"
+)
+
+// GetConfiguration returns the current configuration as a string
+func GetConfiguration() string {
```

Comment:
I think the name here is misleading (I thought this was for use by the program to access configuration values). configuration.AsString() would be more in the go style.